sql/core/src/main/scala/org/apache/spark/status/protobuf/sql/SparkPlanGraphWrapperSerializer.scala

```diff
@@ -53,8 +53,9 @@ class SparkPlanGraphWrapperSerializer extends ProtobufSerDe {
     StoreTypes.SparkPlanGraphNodeWrapper = {
 
     val builder = StoreTypes.SparkPlanGraphNodeWrapper.newBuilder()
-    builder.setNode(serializeSparkPlanGraphNode(input.node))
-    builder.setCluster(serializeSparkPlanGraphClusterWrapper(input.cluster))
+    Option(input.node).foreach(node => builder.setNode(serializeSparkPlanGraphNode(node)))
```

Comment:
Fix the following test in `SQLAppStatusListenerWithRocksDBBackendSuite`:

- `basic`
- `onExecutionEnd happens before onJobEnd(JobSucceeded)`
- `onExecutionEnd happens before multiple onJobEnd(JobSucceeded)s`
- `onExecutionEnd happens before onJobEnd(JobFailed)`
- `onJobStart happens after onExecutionEnd shouldn't overwrite kvstore`
- `handle one execution with multiple jobs`
- `SPARK-32615,SPARK-33016: SQLMetrics validation after sparkPlanInfo updated in AQE`